src/pages/how-to/monitor-system-and-network-activity.mdx

```diff
@@ -1,44 +1,35 @@
 
-# Monitor system and network activity
 
-The activity monitoring feature lets you quickly see what's happening with your network.
-Whether a new machine or user joined your network or the access control policy has been modified, the activity log allows you to track the changes to your network.
+# Monitor System and Network Activity
 
-## Access activity monitoring view
+The activity monitoring functionality in NetBird allows you to observe and track changes to your network infrastructure. This includes events such as when a new machine or user has joined your network, or when an access control policy has been modified.
 
-Activity monitoring is enabled by default for every network, and you can access it in the web UI under the [Activity tab](https://app.netbird.io/activity).
-You can also use the search bar to filter events by activity type.
+## Access the Activity Monitoring View
+
+The activity monitoring feature is enabled by default for every NetBird network. You can access the activity log in the web UI under the [Activity tab](https://app.netbird.io/activity). This view provides a centralized log of network events, and you can use the search bar to filter events by activity type.
 
 <p>
     <img src="/docs-static/img/how-to-guides/activity-monitoring.webp" alt="activity-monitoring" className="imagewrapper-big"/>
 </p>
 
-<Note>
-    The current version of NetBird tracks network changes that occur in the Management server. E.g., changes related to the list of peers, groups, system settings, setup keys, access control, etc.
-    The future versions will support connection events that occur in NetBird agents (e.g., peer A connected to peer B).
-</Note>
+The current version of NetBird tracks network changes that occur in the Management server, such as modifications to the list of peers, groups, system settings, setup keys, and access control policies. Future versions will also support connection events that occur in NetBird agents (e.g., peer A connected to peer B).
```

Comment:
Maybe we should add the whole list as a table?
https://github.com/netbirdio/netbird/blob/5d6dfe59388d43467599603ad4d803f1e0fcd8eb/management/server/activity/codes.go#L15

Reply:
yeah good idea, maybe a summed up version